.github/workflows/pr-check.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
should be '3.12-dev' instead of '3.12'

Reply:
@anthonykim1 Fixed it, thank you for the suggestion.